typescript/infra/src/core/deploy.ts

```diff
@@ -25,6 +25,12 @@ import { DeployEnvironment, RustChainSetup, RustConfig } from '../config';
 import { ConnectionType } from '../config/agent';
 import { writeJSON } from '../utils/utils';
 
+const ENV_NAME_MAP: Record<DeployEnvironment, string> = {
```

Comment:
Check out `deployEnvToSdkEnv` in `typescript/infra/src/config/environment.ts`

Reply:
Adopted.